wsmaster/che-core-api-devfile/src/main/resources/build_docs.sh

```diff
@@ -103,33 +109,73 @@ parse_args() {
                IS_DEPLOY=false
                shift
            ;;
-           --message)
+           --message=*)
                MESSAGE="${i#*=}"
                shift
+           ;;
+           -f=*| --folder=*)
+               FOLDER="${i#*=}"
+               shift
+           ;;
+           --no-clean-up)
+               DO_CLEANUP=false
+               shift
            ;;
             -help|--help)
                print_help
                exit 0
-           ;;
-         esac
+               ;;
+            *)
+                echo "You've passed wrong arg '$key'."
+                echo -e "$HELP"
+                exit 1
+                ;;
+        esac
      done
 }
 
-cleanup
-parse_args "$@"
+copyDocs() {
+  cd $RUN_DIR
+  cp "${TMP_DIR}/docs/index.md" -t ${FOLDER}
+  echo "Docs is saved as ${FOLDER}/index.md"
+}
+
+RUN_DIR=$(pwd)
+TMP_DIR="/tmp/devfile"
 
+DEFAULT_COMMIT_MESSAGE="Update devfile docs"
 COMMIT_MESSAGE=${MESSAGE:-${DEFAULT_COMMIT_MESSAGE}}
+
+DEFAULT_BUILD_DOCKER=false
 IS_DOCKER=${IS_DOCKER:-${DEFAULT_BUILD_DOCKER}}
+
+DEFAULT_DEPLOY=true
 IS_DEPLOY=${IS_DEPLOY:-${DEFAULT_DEPLOY}}
 
-if [[ "$IS_DOCKER" == "true" ]];
-then
-    build_with_docker
-else
-    build_native
+DEFAULT_DO_CLEANUP=true
+DO_CLEANUP=${DO_CLEANUP:-${DEFAULT_DO_CLEANUP}}
+
+parse_args "$@"
+
+if [[ "$IS_DEPLOY" == "true" ]]; then
+  check_github_token_is_set
 fi
+
+cleanupTmpDir
+
+build
+
 if [[ "$IS_DEPLOY" == "true" ]]; then
     deploy
 fi
-cleanup
+
+if [[ "$FOLDER" ]]; then
+    copyDocs
+fi
+
+if [[ "$IS_DEPLOY" == "true" ]]; then
```

Comment:
check this block again, something looks wrong

Reply:
good catch. Thanks!